packages/react-components/src/unstable/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "@fluentui/react-components",
+  "description": "Separate entrypoint for unstable Fluent UI components",
+  "main": "../lib-commonjs/unstable/index.js",
+  "module": "../lib/unstable/index.js",
+  "typings": "../lib/index.d.ts",
```

```suggestion
  "typings": "../lib/unstable/index.d.ts",
```

I guess that typings should also come from the same directory